### DISCUSSION TITLE
Add a guide to modify the CI enviroment

### DISCUSSION THREAD

**Discussion by ghost** [pivpn/pivpn#1579] [Feature requests]
I think it can be useful for those who want to modify the code and add support for new features to the tests (see the addition of Alpine and the lack of a proper test).

**Comment by ghost** [comment]
> I think it can be useful for those who want to modify the code and add support for new features to the tests (see the addition of Alpine and the lack of a proper test).

Tagging currently active contributors: @orazioedoardo @4s3ti @MichaIng @TinCanTech @DerDanilo

**Reply by coolapso** [reply]
Even tho i can see the use of it, and we can definitely add some notes about it, a guide is too much. 

While bash is very welcoming to newbies, CI tools are a bit more complex and hard to grasp and if  you don't know them the guide you need its on the CI Tool documentation :)
